packages/plugin-ext/src/common/plugin-protocol.ts

```diff
@@ -718,6 +722,8 @@ export interface Keybinding {
     mac?: string;
     linux?: string;
     win?: string;
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    args?: any;
```

Comment:
I created a PR to remove the duplicated interface https://github.com/eclipse-theia/theia/pull/9374
Please remove this change.

Reply:
done!